test/pylib/rest_client.py

```diff
@@ -317,13 +317,15 @@ async def backup(self, node_ip: str, ks: str, table: str, tag: str, dest: str, b
                   "snapshot": tag}
         return await self.client.post_json(f"/storage_service/backup", host=node_ip, params=params)
 
-    async def restore(self, node_ip: str, ks: str, cf: str, dest: str, bucket: str, prefix: str, sstables: list[str]) -> str:
+    async def restore(self, node_ip: str, ks: str, cf: str, dest: str, bucket: str, prefix: str, sstables: list[str], scope: str = None) -> str:
         """Restore keyspace:table from backup"""
         params = {"keyspace": ks,
                   "table": cf,
                   "endpoint": dest,
                   "bucket": bucket,
                   "prefix": prefix}
+        if scope is not None:
+            params['scope'] = scope
```

Comment:
Should have a rest api unit test for the additional parameter

Reply:
Dedicated test just for this parameter? And what exactly should it check? 
Presumably [this](https://github.com/scylladb/scylladb/pull/21169#discussion_r1807278357) test would be more than enough no?

Reply:
It can test just the syntax to cement it

Reply:
But if it's fully covered by a functional test that's fine too

Reply:
Hopefully fixed in the recent update